src/Contexts/UserContext.tsx

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+
+interface userTypeContext {
```

Comment:
```suggestion
interface UserContextType {
```

Types generally start with capital letters.

Reply:
Done